go.mod

```diff
@@ -145,10 +145,10 @@ require (
 	code.cloudfoundry.org/garden v0.0.0-20210208153517-580cadd489d2
 	code.cloudfoundry.org/lager v2.0.0+incompatible
 	github.com/CycloneDX/cyclonedx-go v0.8.0
-	github.com/DataDog/appsec-internal-go v1.7.0
+	github.com/DataDog/appsec-internal-go v1.8.0
```

Comment:
Why are we updating this and libddwaf?

Reply:
Transitive dependencies of the new version of dd-trace-go